app/views/govuk_component/metadata.raw.html.erb

```diff
@@ -42,14 +46,16 @@
       </dd>
     <% end %>
     <% if other.present? %>
-      <% other.each do |title, definition| %>
+      <% other.each_with_index do |(title, definition), index| %>
         <%
           definition ||= []
           definition = Array(definition)
         %>
         <% if definition.any? %>
           <dt><%= title %>:</dt>
-          <dd><%= definition.to_sentence.html_safe %></dd>
+          <dd>
+            <%= render file: 'govuk_component/metadata_sentence.raw', locals: { items: definition, toggle_id: index } %>
```

Comment:
Excellent question, I have no idea! If it does I'm not sure what alternative to use. I guess if all of the sections were handled together in a loop (like we originally discussed) then we wouldn't need a partial and there'd be no extra requests. Happy to continue to look into that if you'd like.